pvlib/irradiance.py

```diff
@@ -2151,15 +2156,20 @@ def _dirint_bins(times, kt_prime, zenith, w, delta_kt_prime):
     return kt_prime_bin, zenith_bin, w_bin, delta_kt_prime_bin
 
 
-def dirindex(ghi, ghi_clearsky, dni_clearsky, zenith, times, pressure=101325.,
+@renamed_kwarg_warning(
+    since='11.2',
+    old_param_name='ghi_clearsky',
+    new_param_name='ghi_clear',
+    removal="12.0")
```

Comment:
Any pvlib version string (currently) should be `0.X.Y`.  Note that version strings have a specific format and technical meaning: https://semver.org/.  Additionally, they aren't just for humans to read; they identify versions on e.g. PyPI (see `0.11.1`, not `11.1`, as the latest version here: https://pypi.org/project/pvlib/) so that `pip install pvlib==0.11.1` does the right thing.  So the `0.` is important and cannot be omitted.